Source/vendors/DebugDraw/debug_draw.hpp

```diff
@@ -820,11 +820,11 @@ namespace dd
     static const float FloatEpsilon = 1e-14;
 #endif // FLT_EPSILON && DEBUG_DRAW_USE_STD_MATH
 
-#if defined(M_PI) && DEBUG_DRAW_USE_STD_MATH
-    static const float PI = static_cast<float>(M_PI);
-#else // !M_PI || !DEBUG_DRAW_USE_STD_MATH
+#if defined(HACHIKO_PI) && DEBUG_DRAW_USE_STD_MATH
```

Comment:
why changing this in the vendor code?

Reply:
It's totally me being silly due to the `M_PI` being both defined in some c library and in our project at `Globals.h`. I did the changes and amended the previous commit.